spring-boot-autoconfigure/pom.xml

```diff
@@ -415,6 +415,11 @@
 			<artifactId>aspectjweaver</artifactId>
 			<optional>true</optional>
 		</dependency>
+        <dependency>
+            <groupId>net.sf.ehcache</groupId>
+            <artifactId>ehcache</artifactId>
+            <optional>true</optional>
+        </dependency>
```

Comment:
Check your settings and make sure you are using tabs. This does not seem to be properly indented (and other files in your PR as well).